web-services/client/pom.xml

```diff
@@ -171,11 +171,6 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-api</artifactId>
-            <scope>test</scope>
-        </dependency>
```

Comment:
why are we removing this?  if we're not going to add junit-jupiter-api to each module, why remove it from the ones that already include it?

Reply:
I removed it because it was unused, whereas with the other modules there are tests that rely on the jupiter api. I can add back the unused dependency if you like.

Reply:
i guess what im saying is that this change wasn't required for the work you were doing.  if you're worried about scope creep, well i think this counts as that, right?  only change what you need